app/full_text_search.py

```diff
@@ -369,9 +373,7 @@ def create_document(person):
 
     # Applies two methods because kanji is used in Chinese and Japanese,
     # and romanizing in chinese and japanese is different.
-    romanize_methods = [script_variant.romanize_word_by_unidecode,
-                        script_variant.romanize_japanese_word,
-                        script_variant.romanize_chinese_name]
+    romanize_methods = ROMANIZE_METHODS
```

Comment:
Can you directly use ROMANIZED_METHODS below instead of defining another variable, for simplicity?